examples/ecommerce-netlify-functions/models.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const mongoose = require('mongoose');
+
+const productSchema = new mongoose.Schema({
+  productName: String,
```

Comment:
Rename `productName` -> `name`, `productPrice` -> `price`. No need to preface with "product" since it is part of product schema.